packages/11ty/_includes/web-components/image-sequence/index.js

```diff
@@ -54,9 +63,9 @@ class ImageSequence extends LitElement {
   canvasRef = createRef()
 
   /**
-   * @property bufferReady
+   * @property indexesToBuffer
    * 
-   * Returns an array of indexes to buffer
+   * Returns an array of indexes included in `bufferSize` given `index`
    * 
    **/ 
```

Comment:
Use [JSDoc returns tag](https://jsdoc.app/tags-returns) and `*/` to close doc comments.
```suggestion
   * @returns an array of indexes included in `bufferSize` given `index`
   */ 
```

Reply:
These are incorporated, see https://github.com/thegetty/quire/blob/ec1a79fc19adc1be430c3ea79ae12914079f1928/packages/11ty/_includes/web-components/image-sequence/index.js#L73